host/bcm283x/gpio.go

```diff
@@ -325,28 +328,22 @@ func (p *Pin) FastOut(l gpio.Level) {
 //
 // PWM pins
 //
-// PWM0 is exposed on pins 12, 18 and 40.
+// PWM0 is exposed on pins 12, 18 and 40. However, PWM0 is used for generating
+// clock for DMA and unavailable for PWM.
 //
 // PWM1 is exposed on pins 13, 19, 41 and 45.
 //
-// PWM0 and PWM1 share the same 25Mhz clock source. The period must be a
-// divisor of 25Mhz.
+// PWM1 uses 25Mhz clock source. The period must be a divisor of 25Mhz.
 //
-// Clock pins
-//
-// Clock GPCLK0 is exposed on pins 4, 20 32 and 34.
-//
-// Clock GPCLK2 is exposed on pins 6 and 43.
-//
-// Clocks can only be used with duty 0, gpio.DutyHalf or gpio.DutyMax. They
-// also have relatively limited frequency range from 4.8kHz to 1MHz with a
-// specific set of values supported.
+// DMA driven PWM is available for all pins except PWM1 pins, its resolution is
```

Comment:
Please add: "The number of DMA driven PWM is limited."

Reply:
done.